tests/unit/cli/test_config.py

```
@@ -0,0 +1,216 @@
+        "trigger_time": 1126259462.4,
+        "duration": 4.0,
+        "psd_duration": 1024.0,
+    },
+    "waveform": {"approximant": "IMRPhenomXAS"},
+    "prior": {
+        "M_c": {"type": "uniform", "min": 10.0, "max": 80.0},
+        "q": {"type": "uniform", "min": 0.125, "max": 1.0},
+    },
+    "likelihood": {"f_min": 20.0, "f_max": 1024.0},
+    "sampler": {"type": "flowmc"},
+    "output": {"dir": "/tmp/test"},
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/test"

(S108)